CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 
 * Changed type of prop `tooltipMessage` of `tooltip-decorator` from string to node to allow children.
 
+## Fixes
+
+* Set the `Component.displayName` on all decorators.
```

Comment:
You might want to highlight the fact that this may effect snapshots in peoples repos as they upgrade, to change the component name to something more accurate.